src/test/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoringDemo.java

```diff
@@ -0,0 +1,15 @@
+package com.rakovets.course.java.core.practice.jcf_map;
+
+import com.rakovets.course.java.core.practice.jcf_map.words_monitoring.TextMonitoring;
+
+public class TextMonitoringDemo {
+    public static void main(String[] args) {
+        String text = "hello, I like java, but I don't like java??? skript???.    hello hello, ,      hello";
+
+        System.out.println(TextMonitoring.researchText(text));
+        System.out.println(TextMonitoring.getCountUniqueWords(text));
+        System.out.println(TextMonitoring.getUniqueWords(text));
+        System.out.println(TextMonitoring.getFrequencyWord(text, "word"));
+        System.out.println(TextMonitoring.getFrequencyWords(text, true));
+        }
+    }
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"